person.rb

```diff
@@ -20,6 +21,10 @@ def correct_name
     @name
   end
 
+  def add_rentals(book, date)
```

Comment:
- [ ] kindly make sure the rentals is accessible outside the Person Class, which means you should add getters and setters for the rentals array. Hint `attr_accessor :rentals`